src/main/java/org/codeforamerica/shiba/application/parsers/CountyParser.java

```diff
@@ -37,7 +37,7 @@ public County parse(ApplicationData applicationData) {
                 .map(inputData -> inputData.getValue().get(0))
                 .orElse(pageInputs.get("county").getDefaultValue());
 
-        if (featureFlagConfiguration.get("county-" + County.valueFor(countyName).toString().toLowerCase()) == FeatureFlag.OFF) {
+        if (featureFlagConfiguration.get("county-" + County.valueFor(countyName).name().toLowerCase()) == FeatureFlag.OFF) {
```

Comment:
should this be `.displayName()` rather than `.name()`?

Reply:
So for this we most likely want to keep it the enum value instead. What this does is look for county-stlouis by taking the county enum value and tranforming to a lowercase string. The display name is moreso to view better when reading as a client or caseworker. If we use the displayName it would be 'county-st. louis ' which isnt bad but we would need to handle the space and period. We can change it to that but would require a little more than I am not sure is necessary. Let me know if you feel strongly about it and we can chat

Reply:
Ah got it, I trust your judgement on this!